modules/e2ee/OlmAdapter.js

```diff
@@ -626,3 +646,5 @@ function safeJsonParse(data) {
         return {};
     }
 }
+
+OlmAdapter.events = OlmAdapterEvents;
```

Comment:
Do we use these events outside of the adapter anymore? If not let's get rid of them!

Reply:
Do we still need these then?

Reply:
Yep, it is still used in ManagedKeyHandler